static/sass/_pattern_p-card.scss

```diff
@@ -51,12 +51,13 @@
     @extend %vf-has-box-shadow;
 
     background-color: $color-x-light;
-    border-radius: $spv-inner--x-small;
+    border-radius: 0.125rem;
```

Comment:
whay not use the vanilla default? this is what makes up .p-card:
    @extend %vf-card;
    @extend %vf-is-bordered;
    @extend %vf-has-round-corners;
        padding: calc(#{$spv-inner--large} - 1px);

Reply:
This because these cards are built using grid to achieve what Daniel wanted. I will add `@extend %vf-has-round-corners;`, as is the only one I can use.

Reply:
👍